packages/integrations/src/opnsense/opnsense-integration.ts

```diff
@@ -0,0 +1,207 @@
+import { fetchWithTrustedCertificatesAsync } from "@homarr/certificates/server";
+import { ParseError } from "@homarr/common/server";
+
+import { createChannelEventHistory } from "../../../redis/src/lib/channel";
```

Comment:
Invalid import, please use the redis package directly (if not exported there, just export it)

Reply:
Anyway, I think it's done